src/components/proofs/ProofList.tsx

```diff
@@ -29,8 +31,18 @@ const bottomWrapper = classnames(
 )
 
 export function ProofListSuspended() {
-  const [category, setCategory] = useState<CategoriesTitles>('NFTs')
+  const params = useUrlParams()
+
+  const [category, setCategory] = useState<CategoriesTitles>(
+    params ? 'Email' : 'NFTs'
+  )
   const { hasAnyProof } = useProofStore()
+  if (params) {
```

Comment:
It seems to me that such logic should be in the rendering of sections, and not in the general file

Reply:
Did it